tests/callbacks/callback_settings.py

```diff
@@ -0,0 +1,177 @@
+from typing import Any, Callable, Dict, NamedTuple, Optional, Sequence
+
+import pytest
+
+import composer.callbacks
+import composer.loggers
+import composer.profiler
+from composer import Callback
+from composer.callbacks import EarlyStopper, MemoryMonitor, ThresholdStopper
```

Comment:
clean up these imports by concatentaing all the callback imports together?

Reply:
Done